credentials_manager/lib/credentials_manager/appfile_config.rb

```diff
@@ -144,7 +144,7 @@ def for_lane(lane_name, &block)
     #             be applied.
     def for_platform(platform_name, &block)
       if ENV["FASTLANE_PLATFORM_NAME"] == platform_name.to_s
-        block.call
+        yield
```

Comment:
I believe it doesn't need to be stated unless you want to interact with it directly (like `block.call`). Any Ruby method can `yield` to a block even if no block parameter is stated.